src/vs/workbench/parts/explorers/common/treeExplorer.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+export const VIEWLET_ID_ROOT = 'workbench.view.customTreeExplorer.';
```

Comment:
@octref since you seem to use this always as a prefix, might be more elegant to just expose a function `toViewletId()` that does the concatenation?